app/components/post_list/post_list.js

```diff
@@ -170,15 +167,15 @@ export default class PostList extends PureComponent {
             refreshControl.onRefresh = this.onRefresh;
         }
 
-        const data = this.getPostsWithDates();
         return (
             <FlatList
                 ref='list'
-                data={data}
+                data={postIds}
+                extraData={highlightPostId}
```

Comment:
why do we need this?

Reply:
It's to trigger a re-render if that changes since FlatList is a PureComponent and highlightPostId is needed used when rendering the posts. It's only used for the SearchPreview since `post.highlighted` is no longer set